lib/PutItemBuilder.js

```
@@ -12,9 +12,17 @@ var typeUtil = require('./typeUtil')
  */
 function PutItemBuilder(options) {
   Builder.call(this, options)
+
+  /** @private {string} */
+  this._returnValues = 'ALL_OLD'
 }
 require('util').inherits(PutItemBuilder, Builder)
 
+PutItemBuilder.prototype.setReturnValues = function (val) {
+  this._returnValues = val
```

Should we throw an error for values that we don't support?